service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -163,12 +158,19 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
 
     captureRequestMetrics(request);
     Timer.Sample requestTimerSample = micrometerMetrics.startTimer();
+    Timer.Sample responseTimingSample = micrometerMetrics.startTimer();
```

Comment:
nit: feels like these names should parallel one another. Maybe `TimerSample` is slightly better because of the class name, but as long as they're both the same I'm happy

Reply:
I have updated it to be `responseTimerSample` 👍